src/main/kotlin/com/atlassian/performance/tools/report/table/TabbedAuiTable.kt

```diff
@@ -0,0 +1,107 @@
+package com.atlassian.performance.tools.report.table
+
+internal class AuiTabbedTableFactory(
```

Comment:
In general, I'd avoid factories. They're a wonky construct, which suggests there's something wrong the the object model.
It's not too bad in this case, because it's internal, so I can let it slide.